step-plans/step-plans-simple-yaml-parser/src/test/resources/step/plans/simple/test-simple.plan.yml

```diff
@@ -0,0 +1,27 @@
+name: "test plan"
+root:
+  Sequence:
+    name: Sequence
```

Comment:
@iegorov777 What's the difference between name and nodeName?

Reply:
Seems to be a typo. It should be a nodeName here.